src/modules/form/components/DynamicForm.stories.tsx

```diff
@@ -33,11 +34,23 @@ Default.args = { definition: formDefinition, errors: emptyErrorState };
 
 export const WithWarnIfEmpty = Template.bind({});
 WithWarnIfEmpty.args = {
-  definition: formDefinition,
+  definition: modifyFormDefinition(
+    formDefinition,
+    (item) => (item.warnIfEmpty = true)
+  ),
```

Comment:
@martha this is where I inject the `warnIfEmpty` value, so we don't have to manually add it for every field in the mock form